isa/Makefile

```diff
@@ -94,16 +94,16 @@ endef
 #$(eval $(call compile_template,rv32si,-march=rv32g -mabi=ilp32))
 #$(eval $(call compile_template,rv32mi,-march=rv32g -mabi=ilp32))
 ifeq ($(XLEN),64)
-$(eval $(call compile_template,rv64ui,-march=rv64g -mabi=lp64d))
-#$(eval $(call compile_template,rv64uc,-march=rv64g -mabi=lp64d))
-$(eval $(call compile_template,rv64um,-march=rv64g -mabi=lp64d))
-$(eval $(call compile_template,rv64ua,-march=rv64g -mabi=lp64d))
-$(eval $(call compile_template,rv64uf,-march=rv64g -mabi=lp64d))
-$(eval $(call compile_template,rv64ud,-march=rv64g -mabi=lp64d))
+$(eval $(call compile_template,rv64ui,-march=rv64imafd -mabi=lp64d))
```

Comment:
Don't use `rv64imafd`, use `rv64g` instead, it misses the Zicsr, Zifencei (included in G). This might become an issue with GCC 12.